sites/public/src/pages/listing/ext/[id].tsx

```diff
@@ -81,7 +80,8 @@ export async function getServerSideProps(context: {
 }) {
   let response: AxiosResponse
   try {
-    response = await axios.get(`${process.env.bloomApiBase}/listings/${context.params.id}`, {
+    const extUrl = `${process.env.BLOOM_API_BASE}/listings/${context.params.id}`
```

Comment:
is it intentional here that we are calling the Bloom API instead of using our own table? I thought that there was some discussion to store everything we needed to render the /listings/:id page within Doorway

Reply:
The main goal of the ETL was to be able to filter, sort, and paginate across all listings, so it intentionally only imports enough information to accomplish that goal and return a `base` view for the combined list.  There are some minor tradeoffs either way, but I think it makes the most sense to just query the authoritative data source directly when pulling data for the external listings page.

Reply:
I see... the combined listings table stores much, much more than just what it needs to filter/sort/paginate and this has some issues around data consistency so I'm not sure if this argument holds up, however, it's too much work at this point to change